### ISSUE TITLE
Consider optional zeroize integration

### ISSUE THREAD

**Issue by tmpfs** [kenianbei/vcard_parser#6]
My application that would use this library is dealing with private information and we use [zeroize](https://docs.rs/zeroize/latest/zeroize/) for securely zeroing memory on drop.

For the purposes of the application we need to treat contacts which would be represented by `Vcard` as secret and therefore would like to implement the `Zeroize` trait for `Vcard`.

Are you interested in a PR that adds zeroize support behind a feature flag?

**Comment by kenianbei** [comment]
Yep... I originally created this library just for use in a personal address book tui, but I've got time currently to work on it, so definitely open to any PRs or features you'd like to add.

**Comment by tmpfs** [comment]
Cool @kenianbei , i will take a look at implementing this.

I saw your TUI, it looks pretty cool, will try to take it for a spin one of these days.
